server/server.go

```diff
@@ -217,56 +218,48 @@ func (s *Server) Listeners() (func() error, func() error) {
 	return loop2, loop1
 }
 
-func (s *Server) execQuery(ctx context.Context, compiler *ast.Compiler, txn storage.Transaction, query ast.Body, input ast.Value, explainMode types.ExplainModeV1) (types.QueryResponseV1, error) {
-
-	t := topdown.New(ctx, query, s.Compiler(), s.store, txn).WithInput(input)
-
+func (s *Server) execQuery(ctx context.Context, query string, input ast.Value, explainMode types.ExplainModeV1, includeMetrics bool) (results types.QueryResponseV1, err error) {
```

Comment:
👍  I'm glad to see the rego package being used more widely inside OPA itself. Thanks for doing this!